consul/resource_consul_service_test.go

```diff
@@ -311,12 +314,15 @@ resource "consul_service" "example" {
 	node    = "${consul_node.compute.name}"
 	port    = 80
 	tags    = ["tag0"]
-  }
+	meta    = {
```

Comment:
This goes along with the previous comment. I'm pretty sure understanding this will be enough for me to make the rest of this test actually test my change.